batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/Utils/Utilities.java

```diff
@@ -0,0 +1,33 @@
+package com.lftechnology.batch7crud.Utils;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00120)